app/renderer/components/preferences/payment/advancedSettings.js

```diff
@@ -79,7 +80,17 @@ class AdvancedSettingsContent extends ImmutableComponent {
             className={css(styles.listItem, commonStyles.noMarginBottom)}
             switchClassName={css(styles.checkboxSwitch)}
             labelClassName={css(commonStyles.noMarginBottom)}
-          />
+      />
+
+          {
+            !this.props.ledgerData.getIn(['customer', 'transactions']) ? null
+              : <div className={cx({[css(styles.refundLinkContainer)]: true})}>
+                <a href={this.props.ledgerData.get('refundURL')}
+                  target='_blank'
+                  data-l10n-id='getARefund'
+                  className={cx({[css(styles.refundLink)]: true})} />
```

Comment:
also in BEM style: `<div className={css(styles.refundLinkContainer__refundLink)}>`